lib/prism.rb

```diff
@@ -37,6 +37,27 @@ module Prism
   private_constant :LexCompat
   private_constant :LexRipper
 
+  # Raised when requested to parse as the currently running Ruby version but Prism has no support for it.
+  class CurrentVersionError < ArgumentError
+    # Initialize a new exception for the given ruby version string.
+    def initialize(version)
+      message = +"invalid version: Requested to parse as `version: 'current'`; "
+      gem_version =
+        begin
+          Gem::Version.new(version)
```

Comment:
ruby/ruby CI is failing here because `Gem` is not loaded: https://github.com/ruby/ruby/actions/runs/18561653353/job/52911907413

Reply:
Ugh of course. I'll fix it. Shouldn't have cowboy'd.